app/sass/joint.scss

```diff
@@ -0,0 +1,6 @@
+///////////////////////////////////////////////////////////////////////////////
+// Joint customizations
+///////////////////////////////////////////////////////////////////////////////
+.link-tools .tool-options {
+  display: block;
```

Comment:
Out of curiosity. Is this the best way to show this `.tools-options`?
Shouldn't visible/invisible state be managed via JS?

Reply:
I tried all the ways to activate this by JS but unfortunately I didn't succeed without having to do a major refactoring. For now, this is the simplest way to go :(